src/plugins/intel_npu/src/plugin/npuw/logging.hpp

```diff
@@ -62,3 +62,7 @@ void dump_failure(const std::shared_ptr<ov::Model>& model, const std::string& de
             OPENVINO_THROW("NPUW: Assertion " #expr " failed"); \
         }                                                       \
     } while (0)
+
+#ifdef _MSC_VER
+#    define __PRETTY_FUNCTION__ __FUNCSIG__
```

Comment:
Can be removed if it's not used

Reply:
Let's keep this it can be used for verbosity logs